stores/metadata_test.go

```diff
@@ -2337,6 +2334,83 @@ func TestRenameObjects(t *testing.T) {
 	}
 }
 
+func TestRenameObjectsRegression(t *testing.T) {
+	ss := newTestSQLStore(t, defaultTestSQLStoreConfig)
+	defer ss.Close()
+
+	// define directory structure
+	objects := []string{
+		"/firefly/s1/",
+		"/firefly/s2/",
+		"/suits/s1/",
+		"/lost/",
+		"/movie",
+
+		"/firefly/trailer",
+		"/firefly/s1/ep1",
+		"/firefly/s1/ep2",
+		"/firefly/s2/ep1",
```

Comment:
The naming feels a bit too shady ^^

Reply:
Makes debugging so much easier but ok 😁 I was on the fence whether to rename it last minute myself.